### DISCUSSION TITLE
How to update Swiss on sd2sp2. For beginners.

### DISCUSSION THREAD

**Discussion by SkateLifeAway** [emukidid/swiss-gc#784] [Q&A]
Wanting to know the exact way to update Swiss when new update comes out. Got updates downloaded but not sure if Swiss update is all I need or do I also update everything else like Exploit Under Fire and where do put them on sd card to update.

**Comment by emukidid** [comment]
You don't need to update anything on your memory card. Just put the new uncompressed swiss.dol from the latest release onto the root of your sd2sp2 as boot.dol and it'll be loaded automatically at startup from your older version of Swiss.
